src/Server/Program.fs

```diff
@@ -3,55 +3,34 @@ module ServerCode.Program
 
 open System
 open System.IO
-open Microsoft.AspNetCore.Builder
-open Microsoft.AspNetCore.Hosting
 open Microsoft.Extensions.Logging
-open Giraffe
-open Microsoft.AspNetCore
-open ServerErrors
+open Saturn.Application
 
-let GetEnvVar var = 
+let GetEnvVar var =
     match Environment.GetEnvironmentVariable(var) with
     | null -> None
     | value -> Some value
 
-let getPortsOrDefault defaultVal = 
+let getPortsOrDefault defaultVal =
     match Environment.GetEnvironmentVariable("SUAVE_FABLE_PORT") with
     | null -> defaultVal
     | value -> value |> uint16
 
-
-let errorHandler (ex : Exception) (logger : ILogger) =
-    logger.LogError(EventId(), ex, "An unhandled exception has occurred while executing the request.")
-    clearResponse >=> INTERNAL_ERROR ex.Message
-
-let configureApp db root (app : IApplicationBuilder) =
-    app.UseGiraffeErrorHandler(errorHandler)
-       .UseStaticFiles()
-       .UseGiraffe (WebServer.webApp db root)
-       
-let configureLogging (loggerBuilder : ILoggingBuilder) =
-    loggerBuilder.AddFilter(fun lvl -> lvl.Equals LogLevel.Error)
-                 .AddConsole()
-                 .AddDebug() |> ignore
-
-
-
```

Comment:
@Krzysztof-Cieslak how / where is this replaced with by Saturn?

Reply:
* `UseGiraffe (WebServer.webApp db root)` - is replaced by `router` call in `application` CE
* `UseGiraffeErrorHandler(errorHandler)` - is provided by default by Saturn, can be overriden by using `error_handler` in `application` CE
* `UseStaticFiles` - is replaced by `use_static in `application` CE
* logger configuaration - hmm... good question, I think I haven't added it yet to Saturn. Will put on my TODO list.